src/mkdocs_llmstxt/_internal/plugin.py

```diff
@@ -199,6 +184,96 @@ def on_post_build(self, *, config: MkDocsConfig, **kwargs: Any) -> None:  # noqa
             full_output_file.write_text(full_markdown, encoding="utf8")
             _logger.debug(f"Generated file /{self.config.full_output}.txt")
 
+    def _generate_page_markdown(self, html: str, page: Page) -> str:
+        """Convert HTML to Markdown.
+
+        Parameters:
+            html: The HTML content.
+            page: The page object.
+
+        Returns:
+            The Markdown content.
+        """
+        soup = Soup(html, "html.parser")
+        if self.config.autoclean:
+            autoclean(soup)
+        if self.config.preprocess:
+            _preprocess(soup, self.config.preprocess, str(_get_page_md_path(page)))
+
+        # Convert relative links to absolute links
+        base_uri = self._get_base_url()
+        current_dir = _get_parent_directory(page.file.dest_uri)
+        self._convert_to_absolute_links(soup, base_uri, current_dir)
+
+        return mdformat.text(
+            _converter.convert_soup(soup),
+            options={"wrap": "no"},
+            extensions=("tables",),
+        )
+
+    def _convert_to_absolute_links(self, soup: Soup, base_uri: str, current_dir: str) -> None:
```

Comment:
suggestion: Simplification

Since we don't use `self` anymore, we can make this method a function (by moving it out of the class).